docs/guides/admin/eveapi_admin_contact.md

```diff
@@ -1,22 +1,24 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
-## what
+# Eve API Administrative Contact
+
+## What
 Requests to the EVE API need to have an administrative contact email set before SeAT will queue jobs to process. CCP made the request in [this](https://github.com/eveseat/seat/issues/77) Github issue. To address this, the email address is added to the User-Agent string that is used when making EVE API requests as can be seen [here](https://github.com/eveseat/eveapi/blob/master/src/Helpers/PhealSetup.php#L77).
 
-## why
+## Why
 The error `Failed to queue due to default config` is generated by [this](https://github.com/eveseat/eveapi/blob/master/src/Traits/JobManager.php#L47-L56) check. In order for this check to pass, you need to configure the administrative email address in the SeAT configuration.
 
-## how
+## How
 Adding the email address can be done in two ways. Wither via the command line or via the Web interface.
 
-### command line
+### Command line
 The command `php artisan seat:admin:email` will prompt you to add a valid email address for the administrative contact:
 
 Example:
 
 ![seat admin email](https://i.imgur.com/G1SZuZR.png)
 
-### web interface
+### Web interface
```

Comment:
I would exchange Position of Command Line and Web Interface since we want encourage user to use the Web-Intereface before the Command